usermanager/src/test/java/org/pesmypetcare/usermanager/clients/pet/PetCollectionsManagerClientTest.java

```diff
@@ -0,0 +1,644 @@
+        vaccinationCollectionList.add(new Vaccination(DATE_1, vaccinationData));
+        vaccinationCollectionList.add(new Vaccination(DATE_2, vaccinationData));
+        vaccinationCollectionList.add(new Vaccination(DATE_3, vaccinationData));
+
+        vetVisitCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"reason\": \"It was really pleasant\",\n"
+                + "      \"address\": \"Calle 2, pt. Avenida\"\n"
+                + "    },\n"
+                + "    \"key\": \"1990-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"reason\": \"It was really pleasant\",\n"
+                + "      \"address\": \"Calle 2, pt. Avenida\"\n"
+                + "    },\n"
+                + "    \"key\": \"1995-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "    \"body\": {\n"
+                + "      \"reason\": \"It was really pleasant\",\n"
+                + "      \"address\": \"Calle 2, pt. Avenida\"\n"
+                + "    },\n"
+                + "    \"key\": \"1998-01-08T15:20:30\"\n"
+                + "  }\n"
+                + "]");
+        vetVisitDataJson = new StringBuilder("{\n"
+                + "      \"reason\": \"It was really pleasant\",\n"
+                + "      \"address\": \"Calle 2, pt. Avenida\"\n"
+                + "}"
+        );
+        vetVisitData = new VetVisitData("It was really pleasant", "Calle 2, pt. Avenida");
+        vetVisitCollectionList = new ArrayList<>();
+        vetVisitCollectionList.add(new VetVisit(DATE_1, vetVisitData));
+        vetVisitCollectionList.add(new VetVisit(DATE_2, vetVisitData));
+        vetVisitCollectionList.add(new VetVisit(DATE_3, vetVisitData));
+
+        washCollectionJson = new StringBuilder("[\n"
+                + "  {\n"
+                + "    \"body\": {\n"
+                + "      \"description\": \"It smelt so bad\",\n"
+                + "      \"duration\": 54\n"
+                + "    },\n"
+                + "    \"key\": \"1990-01-08T15:20:30\"\n"
+                + "  },{\n"
+                + "  \"body\": {\n"
+                + "    \"description\": \"It smelt so bad\",\n"
+                + "    \"duration\": 54\n"
+                + "  },\n"
+                + "  \"key\": \"1995-01-08T15:20:30\"\n"
+                + "},{\n"
+                + "  \"body\": {\n"
+                + "    \"description\": \"It smelt so bad\",\n"
+                + "    \"duration\": 54\n"
+                + "  },\n"
+                + "  \"key\": \"1998-01-08T15:20:30\"\n"
+                + "}\n"
+                + "]");
+        washDataJson = new StringBuilder("{\n"
+                + "    \"description\": \"It smelt so bad\",\n"
+                + "    \"duration\": 54\n"
+                + "}"
+        );
+        washData = new WashData("It smelt so bad", 54);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['54' is a magic number.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5549434)